### PR TITLE
fix: handle absence of NATS in mayastor deployment

### PR DESCRIPTION
**Pull request by blaisedias**
Future versions of mayastor will deployed without NATS
Make changes to handle this gracefully.
Note system test should still work for builds
that do use NATS

### CODE REVIEW

**Comment by blaisedias** [comment]
bors try

File: src/common/k8sinstall/util.go [hunk 1]
```diff
@@ -338,6 +340,10 @@ func InstallMayastor() error {
 	}
 
 	for _, yaml := range mayastorYamlFiles {
+		yamlFilepath := filepath.Join(yamlsDir, yaml)
+		if _, err = os.Stat(yamlFilepath); errors.Is(err, os.ErrNotExist) {
+			continue
```
Comment (chriswldenyer) [inline src/common/k8sinstall/util.go:345]: Here and in the other change, might be worth logging the fact that the file is missing and being skipped.

**Review by chriswldenyer — APPROVED** [review]
LGTM

File: src/common/k8sinstall/util.go (hunk 1 above)
Comment (blaisedias) [reply src/common/k8sinstall/util.go:346]: fixed https://github.com/mayadata-io/mayastor-e2e/pull/459/commits/fe9fb90f4d5364b25c50de6ec62a8938530d5a46

**Review by rohan2794 — APPROVED** [review]

**Comment by blaisedias** [comment]
bors merge

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**